code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -429,4 +429,149 @@
 	icon_state = "dominian_noble5"
 	item_state = "dominian_noble5"
 	icon_open = "dominian_noble5_open"
-	icon_closed = "dominian_noble5"
\ No newline at end of file
+	icon_closed = "dominian_noble5"
+
+/obj/item/clothing/suit/storage/toggle/greengov
+	name = "green formal jacket"
+	desc = "A sleek proper formal jacket with gold buttons."
+	icon_state = "suitjacket_green_open"
+	item_state = "suitjacket_green"
+	icon_open = "suitjacket_green_open"
+	icon_closed = "suitjacket_green"
+	body_parts_covered = UPPER_TORSO|ARMS
+
+/obj/item/clothing/suit/storage/toggle/track

```

Comment:
I am sure we already have trackjackets, either use the ones that already exists or replace them fully.